### ISSUE TITLE
Minor bug in SendForm: shipping and currency are transposed

### ISSUE THREAD

**Issue by paulchaplin** [wojodesign/simplecart-js#243]
On lines 1130 and 1131 in simpleCart.js (version 3.0.3), this:

```
currency : simpleCart.shipping()
, shipping : simpleCart.currency().code
```

should be this:

```
currency : simpleCart.currency().code
, shipping : simpleCart.shipping()
```

**Comment by brettwejrowski** [comment]
Thanks @paulchaplin, I will fix this in the next round of updates.

**Comment by brettwejrowski** [comment]
fixed in 3.0.4
